WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -331,8 +332,31 @@ class NotificationsViewController: UIViewController, UIViewControllerRestoration
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: ListTableViewCell.defaultReuseID, for: indexPath)
-        configureCell(cell, at: indexPath)
+        let cell = tableView.dequeueReusableCell(
+            withIdentifier: "NotificationsTableViewCell",
+            for: indexPath
+        )
+
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "NotificationsTableViewCell") as? HostingTableViewCell<NotificationsTableViewCellContent>,
+              let note = tableViewHandler.resultsController?.managedObject(atUnsafe: indexPath) as? Notification else {
+            return UITableViewCell()
+        }
+        cell.host(
+            NotificationsTableViewCellContent(
+                style: .regular(
+                    .init(
+                        title: note.renderSubject()?.string ?? "",
+                        description: note.renderSnippet()?.string,
+                        shouldShowIndicator: !note.read,
+                        avatarStyle: .init(urls: note.allAvatarURLs) ?? .single(note.iconURL!),
+                        actionIconName: nil
+                    )
+                )
+            ),
+            parent: self
+        )
```

Comment:
[Question] I guess we don't handle the "Altered" state yet? Are you planning to work on that in this PR or a separate PR?

Reply:
No I actually should've done it. I'll look into the old code now to see the conditions it is displayed.

Reply:
Resolved: https://github.com/wordpress-mobile/WordPress-iOS/pull/22593/commits/2b1626e887a88d7e87af2fc6b4a598ddafdfdf36